sdlib/dmd/thread.d

```diff
@@ -7,6 +7,7 @@ extern(C):
 // druntime API.
 void thread_suspendAll();
 void thread_resumeAll();
+void thread_scanAll_C(ScanDg* context, typeof(&__sd_scanAllThreadsFn) scan);
```

Comment:
You don't need the `&` here.

Reply:
Not for SDC, but for D you do. Unless you are planning for SDC to be a separate language?